### PR TITLE
Adding recipe r-kdetrees 

### PR DESCRIPTION
**Pull request by eead-csic-compbio**
This PR adds recipe r-kdetrees , created with https://github.com/bgruening/conda_r_skeleton_helper

I am the @eead-csic-compbio user listed in the maintainer section
I run local tests with `python build-locally.py` and `linux64` config.

@conda-forge/staged-recipes @conda-forge/help-r, ready for review!

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by eead-csic-compbio** [comment]
@conda-forge/help-r all test pass. This PR is ready for review.

File: recipes/r-kdetrees/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    of phylogenetic trees based on the methods of Owen and Provan (2011). Such discordant
+    trees may indicate problems with sequence annotation or tree reconstruction, or
+    they may represent interesting biological phenomena, such as horizontal gene transfers.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - eead-csic-compbio
```
Comment (dbast) [inline recipes/r-kdetrees/meta.yaml:51]: Please also add the conda-forge/r team:
```suggestion
    - conda-forge/r
    - eead-csic-compbio  
```

File: recipes/r-kdetrees/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+package:
+  name: r-kdetrees
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:     {{ cran_mirror }}/src/contrib/Archive/kdetrees/kdetrees_{{ version }}.tar.gz
+  sha256: af65c382b3e52a2bfe7ed8b9a3ff05f2cdd0a1a122c972ed2f79d02dae178ede
+
+build:
+  skip: true  # [win]
```
Comment (dbast) [inline recipes/r-kdetrees/meta.yaml:14]: Please switch back to noarch, as the packages has not compiled code and will be build on linux only after the feedstock conversion. So win CI failures can be ignored here.
```suggestion
  noarch: generic
```

File: recipes/r-kdetrees/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+{% set version = '0.1.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-kdetrees
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:     {{ cran_mirror }}/src/contrib/Archive/kdetrees/kdetrees_{{ version }}.tar.gz
```
Comment (dbast) [inline recipes/r-kdetrees/meta.yaml:10]: where does the indention come from here?
```suggestion
  url: {{ cran_mirror }}/src/contrib/Archive/kdetrees/kdetrees_{{ version }}.tar.gz
```

File: recipes/r-kdetrees/meta.yaml (hunk 1 above)
Comment (eead-csic-compbio) [reply recipes/r-kdetrees/meta.yaml:51]: Thanks @dbast, I guess I need user @conda-forge/r to comment on this PR for the tests to pass?

**Comment by eead-csic-compbio** [comment]
> Hi! This is the friendly automated conda-forge-linting service.
> 
> I wanted to let you know that I linted all conda-recipes in your PR (`recipes/r-kdetrees`) and found some lint.
> 
> Here's what I've got...
> 
> For **recipes/r-kdetrees**:
> 
>     * The following maintainers have not yet confirmed that they are willing to be listed here: conda-forge/r. Please ask them to comment on this PR if they are.

Hi @conda-forge/r, can you please comment here so that you can be listed as mantainer? Thanks

**Review by dbast — APPROVED** [review]

**Comment by dbast** [comment]
@eead-csic-compbio thanks and welcome to conda-forge

**Pull request merged**